Add ComboboxMultiple (#79)
vtex/admin-ui: packages/admin-ui-docs/docs/experimental/combobox.mdx

| setLoading    | `React.SetState<boolean>`  | Set loading state                                    |
| setError      | `React.SetState<boolean>`  | Set error state                                      |

### useComboboxMultipleState

It extends the `useComboboxState` props, and:

| Name                | Type       | Description                                      | Required | Default |
| ------------------- | ---------- | ------------------------------------------------ | -------- | ------- |

> | shouldClearOnSelect | `boolean`  | If should clear the text input on select an item | 🚫       | -       |

Reviewer: ```suggestion
| shouldClearOnSelect | `boolean`  | Whether should clear the text input when selecting an item or not | 🚫       | -       |
```
